starboard/doc/networking_performance.md

```diff
@@ -0,0 +1,85 @@
+It is strongly encouraged for platforms to include the
```

Comment:
Please add a bit of a context-setting header here. E.g. `## Overview`, Cobalt 25 implements IETF HTTP/3 and benefits from more specific tuning and customizations to improve its performance and such.

Reply:
Done.